.env.production

```diff
@@ -19,3 +19,4 @@ REACT_APP_GRAPH_API_LOTTERY = "https://api.thegraph.com/subgraphs/name/pancakesw
 
 REACT_APP_SNAPSHOT_BASE_URL = "https://hub.snapshot.org"
 REACT_APP_SNAPSHOT_VOTING_API = "https://voting-api.pancakeswap.info/api"
+REACT_APP_SNAPSHOT_VOTING_API = ""
```

Comment:
Was it supposed to be `REACT_APP_GRAPH_API_NFTMARKET`?

Reply:
😅